code/modules/mob/living/carbon/human/species/_species.dm

```diff
@@ -515,6 +515,8 @@
 /datum/species/proc/disarm(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style)
 	if(user == target)
 		return FALSE
+	if(IS_HORIZONTAL(user))
+		return FALSE
```

Comment:
this will prevent people who have martial arts from doing combos involving disarms while downed aswell.

Reply:
should be fixed now